CONTRIBUTING.adoc

```diff
@@ -214,6 +217,12 @@ Once your author file is defined, you can return to your blog post file
 (`1970-01-01-hello-world.adoc`), finish creating the "front matter" and then
 write your blog post!
 
+The `opengraph` section is optional. It allows you to define a preview of
```

Comment:
It's optional, but if not provided `images/logo-title-opengraph.png` is used :-)